features/mobile/posts_from_main_page.feature

```diff
@@ -21,8 +21,10 @@ Feature: posting from the mobile main page
   Scenario: post and delete some text
     Given I visit the mobile publisher page
     And I append "I am eating yogurt" to the publisher
-    And I select "Unicorns" from "aspect_ids_"
-    And I press "Share"
+    And I press the aspect dropdown
+    And I toggle the aspect "Unicorns"
```

Comment:
Could you please add a test where you post to two different aspects and check that the post is visible in these two and only these two?

Reply:
Oui.

Reply:
@Flaburgan Can it be that there is no test for setting more than one aspect at same time? Even in Desktop-Mode?

Reply:
They are: https://github.com/diaspora/diaspora/blob/develop/features/desktop/posts_from_main_page.feature#L58

Reply:
Help requested - cucumber tests still won't work. I don't get test to work - the Aspects button will (in test) not expand, or aspects are not 'visible' However - on real browser everything seems to work.